tests/Maker/MakeEntityTest.php

```diff
@@ -487,6 +487,37 @@ public function getTestDetails()
             ->configureDatabase(false),
         ];
 
+        if (version_compare(PHP_VERSION, '7.4', '>=')) {
```

Comment:
```suggestion
        if (\PHP_VERSION_ID >= 70400) {
```

like used in other tests